src/modify.js

```diff
@@ -4,7 +4,7 @@ const lock = new (require('rwlock'))()
 const ioOperations = require('./io-operations.js')
 const config = require('./config').config
 
-const isImportMap = config && config.manifestFormat === 'import-map'
+const isImportMap = config && (config.manifestFormat === 'import-map' || config.manifestFormat === 'importmap')
```

Comment:
@joeldenning use of import-map and importmap, backward compatibility I guess? would it be better if we choose one?

Reply:
Yeah I agree that choosing one is better - importmap makes more sense to me because that’s how it is in script tags in the spec.

But yeah I’m keeping around import-map for now for backwards compatibility — I think @frehner at Jump is using that

Reply:
we could update ours if we need to, not a big deal to me.

Reply:
👍 updated to require `importmap` instead of `import-map`